01-basics/00-setup.R

```diff
@@ -0,0 +1,22 @@
+# install
+if(!require('devtools')) install.packages("devtools")
+if(!require('tidyverse')) install.packages("tidyverse")
+if(!require('natverse')) devtools::install_github("natverse/natverse")
+if(!require('lhns')) devtools::install_github("jefferislab/lhns")
+
+# load natversee functions
+library(natverse)
+library(tidyverse)
+library(lhns)
+## What is lhns? It is a little data package for lateral horn neurons, made for Frechter et al. 2019, eLife.
+## It contains some data we will play with in the next R file
+
+# set working directory to location of this file
+try(setwd(dirname(attr(body(function() {}),'srcfile')$filename)))
+
+# load data from previoous sessions
+if (exists(".RData")){
+  load(".RData")
```

Comment:
@alexanderbates I'm not sure this is a good idea – you can get all kinds of stuff in these session files by mistake. And .RData files are invisible by default in the Finder. If it's not essential for some reason I would remove.

Reply:
Okay, I have removed these lines